src/azure-cli/setup.py

```diff
@@ -149,7 +149,8 @@
     'websocket-client~=0.56.0',
     'xmltodict~=0.12',
     'javaproperties==0.5.1',
-    'jsondiff==1.2.0'
+    'jsondiff==1.2.0',
+    'tabulate==0.8.3'
```

Comment:
I thought we will fix it in knack, or am I missing anything?

Reply:
Fix in knack is a better solution, so we don't need to put a tabulate version wherever knack appears. The fix here is just for a quick solution within our control. Since tabulate released 0.8.5 and fixed their issue, we could abandon the quick fix here and do it in knack.

Reply:
Agree.